File: examples/game-server-demo-1/GameServerDemo1/src/main/java/com/huya/ig/game/logic/Player.java
package com.huya.ig.game.logic;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * @author wangpeng1@huya.com
 * @date 2020/7/20
 * @description
 */
@Data
@AllArgsConstructor
public class Player {

    private String uid;
    private int score;
    private String nick;
    private String avatar;

}

File: examples/game-common-demo-a/server-python/src/logic/player.py
import json

class Player(object):
    """ 玩家对象 """
    uid = None
    score = 0
    nick = None
    avatar = None

    def __init__(self, uid):
        self.uid = uid

    def toDict(self):
        """ 转成字典 """
        player= {}
        player['uid'] = self.uid
        player['score'] = self.score
        player['nick'] = self.nick
        player['avatar'] = self.avatar
        return player

class PlayerEncoder(json.JSONEncoder): 
  def default(self, obj): 
    if isinstance(obj, Player): 
        return obj.toDict()
    return json.JSONEncoder.default(self, obj) 